src/components/Shifting/BoardView.tsx

```diff
@@ -35,16 +36,16 @@ export default function BoardView() {
   });
 
   const [modalFor, setModalFor] = useState<{
-    externalId?: string;
+    external_id?: string;
```

Comment:
Why the casing change? 🤔

Reply:
The **IShift** interface has that property with lowercase, so I made that change to maintain consistency.
Is it okay or I should revert it back?
![image](https://github.com/user-attachments/assets/ffd24b10-fef1-46a9-bbff-506852430377)